pom.xml

```
@@ -37,8 +37,8 @@
   </scm>
 
   <properties>
-    <revision>1.020</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>1</revision>
```

```suggestion
    <revision>2</revision>
```
1.0x is considerable newer than 1, requiring an increment by one.